staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -310,6 +310,9 @@ func (o *DrainCmdOptions) RunDrain() error {
 				fmt.Fprintf(o.ErrOut, "error: unable to drain node %q due to error:%s, continuing command...\n", info.Name, err)
 				continue
 			}
+			fmt.Fprintf(o.ErrOut, "DEPRECATED WARNING: Aborting the drain command in a list of nodes will be deprecated.\n"+
```

Comment:
Do we know what version?

Reply:
> Do we know what version?

?

Reply:
Sorry should have been more clear.

Is there a targeted Kubernetes version for this change that we can include in the deprecation warning?

Reply:
> Sorry should have been more clear.
> 
> Is there a targeted Kubernetes version for this change that we can include in the deprecation warning?
```
Add deprecated message in v1.22
Set ignore errors default to true, and remove --ignore-errors flag in v1.23
```
Add version plan in message?

Reply:
Yes please. If we know the plan it's a good way to communicate that to users.

Reply:
> DEPRECATED WARNING: Aborting the drain command in a list of nodes is deprecated in v1.22 and will be removed in v1.23

Reply:
In v1.22 just add an DEPRECATED WARNING message, it will be deprecated and removed in v1.23, so may be just
```
DEPRECATED WARNING: Aborting the drain command in a list of nodes will be deprecated in v1.23.
```